### DISCUSSION TITLE
How to use system clipboard?

### DISCUSSION THREAD

**Discussion by eagle-k** [nix-community/NixOS-WSL#190] [Q&A] [answered]
Is there a way to access the Windows clipboard?

The following method using `clip.exe` did not work.

```bash
$ echo hello > hello.txt

$ cat hello.txt
hello

$ cat hello.txt | /mnt/c/Windows/system32/clip.exe
```

**Comment by 573** [comment, marked as answer]
can you try making https://github.com/573/winbin/blob/master/pbcopy executable and then `echo hello | pbcopy` ?
Set PATH accordingly.

**Reply by eagle-k** [reply]
This seems to work.
Thank you!
